llvm/lib/CodeGen/SelectionDAG/FastISel.cpp

```diff
@@ -1148,9 +1148,11 @@ bool FastISel::lowerCall(const CallInst *CI) {
   CLI.setCallee(RetTy, FuncTy, CI->getCalledOperand(), std::move(Args), *CI)
       .setTailCall(IsTailCall);
 
-  diagnoseDontCall(*CI);
-
-  return lowerCallTo(CLI);
+  if (lowerCallTo(CLI)) {
+    diagnoseDontCall(*CI);
+    return true;
+  } else
+    return false;
```

Comment:
```suggestion
  }
  
  return false;
```

No else after return